libcylinder/tests/mod.rs

```diff
@@ -21,6 +21,22 @@ use cylinder::{Context, PrivateKey, Signature};
 
 type TestResult = Result<(), Box<dyn std::error::Error>>;
 
+/// Verifies that the round-trip of signing and verifying some bytes using a signer and verifier
+/// produced using the given context and private key.
+fn test_signing_and_verification<C: Context + 'static>(
```

Comment:
This is a good test, can you add the test run to the secp256k1 tests, too?

Reply:
I have it ready in a follow-up PR